artifacts/deploy/200_rbac.yaml

```diff
@@ -146,7 +146,7 @@ rules:
   - apiGroups:
     - admission.autoscaling.openshift.io
     resources:
-    - clusterresourceoverrides
+    - clusterresourceoverride
```

Comment:
Was there a particular reason you "depluralized" this ? I don't think it should technically matter, I think it may work both ways, but the convention is usually to use the plural.

Reply:
I think there was some rbac permissions error when i tried to use `clusterresourceoverrides`, but when I depluralized it, it disappeared. Although that probably means, there's some other issue somewhere else, but I'll try it again "repluralized" and see what happens.

Reply:
Hmm, looking through the rbac, we probably only need one group that does this, there is another one further up here: 
https://github.com/openshift/cluster-resource-override-admission-operator/blob/68ab8617ad0fa5e4527f3055159299131b822d95/artifacts/deploy/200_rbac.yaml#L91-L97

And this should probably come out because we aren't using hostnetwork anymore: 
https://github.com/openshift/cluster-resource-override-admission-operator/blob/68ab8617ad0fa5e4527f3055159299131b822d95/artifacts/deploy/200_rbac.yaml#L135-L145

There might be corresponding chunks in the csv, I haven't looked yet.

Reply:
Yeah, I see this if I revert it:
```
E0618 23:27:27.191713 1 worker.go:67] error syncing '/cluster': resource=clusterroles failed to ensure RBAC - failed to create ClusterRole - clusterroles.rbac.authorization.k8s.io "system:clusterresourceoverride-requester" is forbidden: user "system:serviceaccount:clusterresourceoverride-operator:clusterresourceoverride-operator" (groups=["system:serviceaccounts" "system:serviceaccounts:clusterresourceoverride-operator" "system:authenticated"]) is attempting to grant RBAC permissions not currently held:
{APIGroups:["admission.autoscaling.openshift.io"], Resources:["clusterresourceoverride"], Verbs:["create"]} &ClusterRole{ObjectMeta:{system:clusterresourceoverride-requester 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[operator.autoscaling.openshift.io/clusterresourceoverride:true] map[] [{operator.autoscaling.openshift.io/v1 ClusterResourceOverride cluster b9ae5574-351b-4fc3-b554-59d11d349a59 0xc00087bc09 0xc00087bc08}] [] []},Rules:[]PolicyRule{PolicyRule{Verbs:[create],APIGroups:[admission.autoscaling.openshift.io],Resources:[clusterresourceoverride],ResourceNames:[],NonResourceURLs:[],},},AggregationRule:nil,}, requeuing
```